app-emulation/cloud-init/cloud-init-9999.ebuild

```diff
@@ -33,15 +33,15 @@ CDEPEND="
 	dev-python/requests[${PYTHON_USEDEP}]
 	dev-python/jsonpatch[${PYTHON_USEDEP}]
 	dev-python/jsonschema[${PYTHON_USEDEP}]
-	dev-python/six[${PYTHON_USEDEP}]
 "
 DEPEND="
 		${CDEPEND}
 	test? (
 		>=dev-python/httpretty-0.7.1[${PYTHON_USEDEP}]
 		dev-python/mock[${PYTHON_USEDEP}]
-		dev-python/nose[${PYTHON_USEDEP}]
-		dev-python/coverage[${PYTHON_USEDEP}]
+		dev-python/pytest[${PYTHON_USEDEP}]
+		dev-python/pytest-cov[${PYTHON_USEDEP}]
```

Comment:
pytest-cov should be avoided for us if possible as, like coverage, given it doesn't add any value for us when figuring out if something actually _works_ or not on e.g. an odd architecture: https://projects.gentoo.org/python/guide/pytest.html?highlight=coverage#avoiding-dependencies-on-other-pytest-plugins.

Reply:
Understood, will update with my findings soon.